mkdocs.yml

```diff
@@ -114,6 +114,9 @@ extra_javascript:
 # Page tree
 nav:
   - Home: index.md
+  - Use Case - Manufacturer: Use Case - Device Manufacturer.md
+  - Use Case - NGO: Use Case - NGO.md
+  - Use Case - PAYGO: Use Case - Paygo Entrepreneur.md
```

Comment:
Maybe something like `Use cases` as the folder and the the links below?

```
- Use cases:
    Manufacturer: Use Case - Device Manufacturer.md
    NGO: Use Case - NGO.md
```

Could you then, also re-arrange the files in the folder structure?